releasenotes/notes/0.2/gate-cutting-workflow-ebbedbdb1313b258.yaml

```diff
@@ -32,7 +32,7 @@ issues:
     The :mod:`~circuit_knitting_toolbox.circuit_cutting` package only supports :class:`~qiskit.quantum_info.PauliList` observables for calculating expectation values. Support for calculating
     expectation values for more observable types, including :class:`~qiskit.quantum_info.SparsePauliOp`, is expected no sooner than 0.3.0.
   - |
-    The ``Sampler``\ s from Qiskit and Qiskit Aer do not support mid-circuit measurements in statevector mode. For more on generating exact quasi-distributions using the :class:`~qiskit.primitives.BaseSampler` interface, check out our `how-to guide <https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/tree/main/docs/circuit_cutting/how-tos/how_to_generate_exact_quasi_dists_from_sampler.ipynb>`_.
+    The ``Sampler``\ s from Qiskit and Qiskit Aer do not support mid-circuit measurements in statevector mode. For more on generating exact quasi-distributions using the :class:`~qiskit.primitives.BaseSampler` interface, check out our `how-to guide <https://github.com/Qiskit/qiskit-addon-cutting/blob/stable/0.7/docs/circuit_cutting/how-tos/how_to_generate_exact_quasi_dists_from_sampler.ipynb>`_.
```

Comment:
Technically this note is for 0.2, but I linked to 0.7 so it's more recent. Lmk if I should link to 0.9. The most important thing is to use a stable branch because `main` will soon-ish not have non-API docs